tidb-lightning/tidb-lightning-physical-import-mode.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Physical Import Mode
+---
+
+# Physical Import Mode
+
+Physical import mode is an efficient and fast import mode that inserts data directly into TiKV ndoes as key-value pairs without going through the SQL interface. It is suitable for importing up to 100 TB of data.
+
+Before you use physical import mode, make sure to read [Requirements and restrictions](#requirements-and-restrictions).
+
+## Implementation
+
+1. Before importing data, TiDB Lightning automatically switches the TiKV nodes to "import mode", which improves write performance and stops PD scheduling and auto-compaction.
```

Comment:
For this step, the latest version is https://github.com/pingcap/docs-cn/blob/master/tidb-lightning/tidb-lightning-physical-import-mode.md#%E5%8E%9F%E7%90%86%E8%AF%B4%E6%98%8E for v6.2. The updates in https://github.com/pingcap/docs-cn/pull/10522 will be aligned in another PR, right?

Reply:
Yes. It will be aligned in a v6.2 PR.